core/src/main/java/org/springframework/security/aot/hint/CoreSecurityRuntimeHints.java

```diff
@@ -19,14 +19,18 @@
 import java.util.List;
 import java.util.stream.Stream;
 
+import org.springframework.aop.SpringProxy;
```

Comment:
@sdeleuze Should we move SpringProxy, Advised and DecoratingProxy to the Spring Framework ? They can be useful for other